data/npc/lib/npcsystem/modules.lua

```diff
@@ -700,7 +700,7 @@ if Modules == nil then
 				i = i + 1
 			end
 
-			if name ~= nil and container ~= nil and itemid ~= nil and cost ~= nil then
+			if name and container and itemid and cost then
```

Comment:
@TheSumm oh, i see, actually this code is legacy, i can't have any idea what was/is it purposes, maybe some really old tfs version had a usage? this npc system needs to be completely reworked, i'd say 1.3 roadmap, i'll left as is, the purpose os this pr is make code looks more consistent in datapack not fix anything, anyway well noticed.